catalyst/callbacks/scheduler.py

```diff
@@ -115,9 +115,12 @@ def _scheduler_step(
     ):
         if isinstance(scheduler, torch.optim.lr_scheduler.ReduceLROnPlateau):
             scheduler.step(reduced_metric)
-            lr = scheduler.optimizer.param_groups[0]["lr"]
```

Comment:
Hi @BloodAxe 
Unfortunately, `torch.optim.lr_scheduler.ReduceLROnPlateau` doesn't inherit from `_LRScheduler`, so we cannot use neither `get_last_lr()` nor `get_lr()` for it